<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuConnexion</class>
 <widget class="QDialog" name="MenuConnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connexion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/img/connexion.png</normaloff>:/img/connexion.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelNamePlayer">
     <property name="text">
      <string>Nom du joueur :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditNamePlayer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelIPAddress">
     <property name="text">
      <string>Adresse IP :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditIPAddress">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string>Port :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="text">
      <string>1234</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="buttonConnexion">
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="buttonAnnuler">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditNamePlayer</tabstop>
  <tabstop>lineEditIPAddress</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>buttonConnexion</tabstop>
  <tabstop>buttonAnnuler</tabstop>
 </tabstops>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
